code/modules/projectiles/guns/ds13/divet.dm

```diff
@@ -138,20 +134,13 @@ Projectile logic
 	armor_penetration = 15
 	icon_state = "divet_ap"
 
+//Mostly normal rounds with a little extra armor pen, but they also set you on fire
 /obj/item/projectile/bullet/ls_slug/incendiary
-	damage_type = BURN
```

Comment:
the damage type being burn is fine, they still use ballistic armor for blocking purposes